packages/waku/src/lib/plugins/vite-plugin-rsc-hmr.ts

```diff
@@ -41,7 +42,7 @@ if (import.meta.hot && !globalThis.__WAKU_HMR_CONFIGURED__) {
     document.head.appendChild(script);
     // avoid HMR flash by first applying the new and removing the old styles 
     if (style) {
-      queueMicrotask(style.remove);
+      queueMicrotask(() => style.parentElement?.removeChild(style));
```

Comment:
I think you want this?
```suggestion
      queueMicrotask(() => style.remove());
```

Reply:
Nope this errors out sometimes. Illegal invocation.

Reply:
I see...